model/database/common.yaml

```diff
@@ -21,3 +21,24 @@ groups:
           The `error.type` SHOULD match the error code returned by the database or the client library,
           the canonical name of exception that occurred, or another low-cardinality error identifier.
           Instrumentations SHOULD document the list of errors they report.
+
+  - id: attributes.db.cosmos.minimal
+    type: attribute_group
+    brief: 'Cosmos DB attributes'
+    attributes:
+      - ref: db.system
+        requirement_level: required
```

Comment:
Sample in PR description seems off - cosmosdb should have some prefix like Azure or other namespaces?

Reply:
dimension names are same as attributes names in our traces. So following same convention